uqcsbot/api.py

```diff
@@ -56,16 +63,27 @@ def __call__(self, **kwargs) -> dict:
 
         Attempts to retry the API call if rate-limited.
         """
-        fn = partial(self._client.api_call, self._method, **kwargs)
+        def get_fn(call_type):
+            client = getattr(self, f'_{call_type}_client')
```

Comment:
Will this always be exactly user client or bot client? I'm not a python main, but this seems more magic than it needs to be